File: Week_5/problem_3-Max_and_Min.py
# https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/G

# Function to find the maximum number
def max_num(ar):
    max_val = float('-inf')     # same as INT_MAX of c or cpp

    for num in ar:
        if num > max_val:
            max_val = num
    return max_val

# Function to find the minimum number
def min_num(ar):
    min_val = float('inf')      # same as INT_MIN of c or cpp

    for num in ar:
        if num < min_val:
            min_val = num
    return min_val

# Reading input
n = int(input())
ar = list(map(int, input().split()))

# Finding max and min
max_val = max_num(ar)
min_val = min_num(ar)

print(min_val, max_val)

File: Week_5/problem_3-Max_and_Min.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/223205/problem/G

#include <bits/stdc++.h>
using namespace std;

int max_num(int ar[], int n)
{
    int max = INT_MIN;
    for (int i = 0; i < n; i++)
    {
        if (ar[i] > max)
            max = ar[i];
    }
    return max;
}

int min_num(int ar[], int n)
{
    int min = INT_MAX;
    for (int i = 0; i < n; i++)
    {
        if (ar[i] < min)
            min = ar[i];
    }
    return min;
}

int main()
{
    int n;
    cin >> n;
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        cin >> ar[i];
    }

    int max = max_num(ar, n);
    int min = min_num(ar, n);
    cout << min << " " << max;

    return 0;
}
